ui/voice/@build/lexicon/moves-vn-lex.json

```diff
@@ -0,0 +1,86 @@
+  { "in": "f", "tok": "f", "tags": [ "file", "move" ] },
+  { "in": "g", "tok": "g", "tags": [ "file", "move" ] },
+  { "in": "h", "tok": "h", "tags": [ "file", "move" ] },
+  { "in": "một", "tok": "1", "tags": [ "rank", "move", "choice" ] },
+  { "in": "hai", "tok": "2", "tags": [ "rank", "move", "choice" ] },
+  { "in": "ba", "tok": "3", "tags": [ "rank", "move", "choice" ] },
+  { "in": "bốn", "tok": "4", "tags": [ "rank", "move", "choice" ] },
+  { "in": "năm", "tok": "5", "tags": [ "rank", "move", "choice" ] },
+  { "in": "sáu", "tok": "6", "tags": [ "rank", "move", "choice" ] },
+  { "in": "bảy", "tok": "7", "tags": [ "rank", "move", "choice" ] },
+  { "in": "tám", "tok": "8", "tags": [ "rank", "move", "choice" ] },
+
+  { "in": "tốt", "tok": "P", "tags": [ "role", "move" ] },
+  { "in": "ngựa", "tok": "N", "tags": [ "role", "move" ] },
+  { "in": "mã", "tok": "N", "tags": [ "role", "move" ] },
+  { "in": "tượng", "tok": "B", "tags": [ "role", "move" ] },
+  { "in": "tịnh", "tok": "B", "tags": [ "role", "move" ] },
+  { "in": "xe", "tok": "R", "tags": [ "role", "move" ] },
+  { "in": "hậu", "tok": "Q", "tags": [ "role", "move" ] },
+  { "in": "vua", "tok": "K", "tags": [ "role", "move" ] },
+
+  { "in": "nhập thành", "val": "castle", "tags": [ "move" ] },
+  { "in": "nhập thành gần", "val": "O-O", "tags": [ "move" ] },
+  { "in": "nhập thành cánh vua", "val": "O-O", "tags": [ "move" ] },
+  { "in": "nhập thành xa", "val": "O-O-O", "tags": [ "move" ] },
+  { "in": "nhập thành cánh hậu", "val": "O-O-O", "tags": [ "move" ] },
+  { "in": "ăn", "val": "x", "tags": [ "move" ] },
+  { "in": "phong", "val": "=", "tags": [ "move" ] },
+
+  { "in": "hoãn", "val": "takeback", "tags": [ "command" ] },
+  { "in": "khoan", "val": "takeback", "tags": [ "command" ] },
+  { "in": "hoà", "val": "draw", "tags": [ "command", "round" ] },
+  { "in": "xin hoà", "val": "draw", "tags": [ "command", "round" ] },
+  { "in": "chấp nhận", "val": "draw", "tags": [ "command", "round" ] },
+  { "in": "thua", "val": "resign", "tags": [ "command", "round" ] },
+  { "in": "xin thua", "val": "resign", "tags": [ "command", "round" ] },
+  { "in": "chơi lại", "val": "rematch", "tags": [ "command", "round" ] },
+  { "in": "ván nữa", "val": "rematch", "tags": [ "command", "round" ] },
+  { "in": "xoay", "val": "flip", "tags": [ "command" ] },
+  { "in": "đồng hồ", "val": "clock", "tags": [ "command", "round" ] },
+  { "in": "đối thủ", "val": "who", "tags": [ "command", "round" ] },
+  { "in": "tiếp theo", "val": "next", "tags": [ "command", "puzzle" ] },
+  { "in": "tiếp", "val": "next", "tags": [ "command", "puzzle" ] },
+  { "in": "tốt", "val": "upvote", "tags": [ "command", "puzzle" ] },
+  { "in": "thích", "val": "downvote", "tags": [ "command", "puzzle" ] },
+  { "in": "không thích", "val": "solve", "tags": [ "command", "puzzle" ] },
+  { "in": "giúp đỡ", "val": "help", "tags": [ "command" ] },
+  { "in": "dừng", "val": "stop", "tags": [ "command" ] },
```

Comment:
Stop (along with yes & no) is critical in timer disambiguation. I assume both it and yes being dừng is a typo

Reply:
sorry just realized the accents are different. does it change the pronunciation a great deal so that they can be easily distinguished?

Reply:
yeah, the pronunciation of `dừng` and `đúng are very different.